image_generation/requirements.txt

```diff
@@ -0,0 +1,5 @@
+--extra-index-url https://download.pytorch.org/whl/cpu
+torch==2.2.2+cpu; sys_platform != 'darwin'
+torch==2.2.2; sys_platform == 'darwin'
```

Comment:
Please, add a comment explaining the diff

Reply:
Fixed